<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>502</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QFrame" name="frame">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_6">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_4">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_5">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>Ninguno</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Negro</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cafe</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rojo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Naranja</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Amarillo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Verde</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Azul</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Violeta</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gris</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blanco</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <item>
           <property name="text">
            <string>Ninguno</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Negro</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cafe</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rojo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Naranja</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Amarillo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Verde</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Azul</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Violeta</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gris</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blanco</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_6">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Ninguno</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Negro</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cafe</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rojo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Naranja</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Amarillo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Verde</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Azul</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Violeta</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gris</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blanco</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_3">
          <item>
           <property name="text">
            <string>Ninguno</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Negro</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cafe</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rojo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Naranja</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Amarillo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Verde</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Azul</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Violeta</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gris</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blanco</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dorado</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Plata</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_4">
          <item>
           <property name="text">
            <string>Ninguno</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cafe</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rojo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Amarillo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Verde</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Azul</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Violeta</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gris</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dorado</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Plata</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_5">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Ninguno</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cafe</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rojo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Naranja</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Amarillo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Verde</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Azul</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Violeta</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gris</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>5 / 6 bandas</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>491</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Banda 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Banda 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Banda 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Multiplicador</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Tolerancia</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Temperatura</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
